package.yaml

```diff
@@ -52,8 +52,8 @@ default-extensions:
 dependencies:
 - base >= 4.7 && < 5
 - QuickCheck >= 2.6 && < 2.15
-- fakedata
-- random
+- fakedata >= 0.6.0 && < 1.1.0
+- random  >= 1 && < 1.3.0
```

Comment:
While I'm not opposed to this, having relaxed bounds makes maintaining things in Stackage easier. Also is there any reason on why we don't want to support older `random` package ?

Reply:
this isn't really my intention, I mucked up merging.

Reply:
No worries, thank you!